.vib/wordpress/cypress/cypress/support/commands.js

```diff
@@ -20,7 +20,7 @@ Cypress.Commands.add("login", (
 ) => {
   cy.clearCookies();
   cy.visit('/wp-login.php')
-  cy.get('#user_login').type(username);
-  cy.get('#user_pass').type(password);
-  cy.get('#wp-submit').click();
-});
\ No newline at end of file
+  cy.get('#user_login').should('not.be.disabled').type(username);
+  cy.get('#user_pass').should('not.be.disabled').type(password);
+  cy.get('#wp-submit').should('be.visible').click();
```

Comment:
Hmm I think that we should not be performing assertion to check that a given functionality works in `commands.js`, but rather create a test case if that is what we intend to. 

In fact, we are already verifying the same in:
https://github.com/bitnami/charts/blob/bea8ff264dc62115a606a3ea4c8b23efd9275834/.vib/wordpress/cypress/cypress/integration/wordpress_spec.js#L39-L40

Reply:
I agree, but I'm afraid this is needed not for testing purposes but flakiness avoidance. I've seen the tests fail because the input field can't be typed into because it has been marked as disabled. This check helps with this case.